### PR TITLE
Form

### PR DESCRIPTION
**Pull request by TaiwoEnoch**
## I create this PR to implement blog app form requirements

# Project requirements
- [ ] Create a method called current_user in your ApplicationController to make current user data available to all controllers.
  - [ ] It will return the first user from the database.
  - [ ] Don't worry about proper authentication at this time. You will add it in the upcoming projects.
- [ ] Create forms to perform the following functions:
  - [ ] Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
  - [ ] Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- [ ] Allow Users to add likes to Posts.

### CODE REVIEW

File: app/views/posts/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<h2>Add new post</h2>
+<%= form_with model: @post, url: user_posts_path(@user) do |f| %>
+  <div class="form-container">
+  <label for="post_title">title:</label>
+    <%= f.text_field :title, required: true, autofocus: true, class: "form-input", placeholder: "Enter post the title" %>
+    <label for="post_title">text:</label>
+    <%= f.text_area :text, required: true, autofocus: true, class: "form-input", placeholder: "Enter post the body"%>
+    <%= f.submit "Create", class: "create-btn" %>
+  </div>
+<% end %>
```
Comment (Ismailco) [inline app/views/posts/new.html.erb:12]: - [ ] [OPTIONAL] This form is not displayed in the views kindly consider including it in the user show page so users can click a button and redirect to it to create a new post

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @TaiwoEnoch, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
- [ ] There should be a form in the views/comments/new.html.erb file which accepts input for creating comments. Kindly consider implementing this so users can add a comment to the post

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TaiwoEnoch** [comment]
Please I will implement the optional suggestions after i get the project approved, have to get it approved so I can have more time to Implement the changes with being on pressure like I am right now. Thanks for your suggestion

File: app/views/posts/new.html.erb (hunk 1 above)
Comment (TaiwoEnoch) [reply app/views/posts/new.html.erb:12]: Thanks a lot for your optional suggestion @Ismailco  I will implement it once i get the code approved.

**Review by Owusu-Desmond — APPROVED** [review]
Hi @TaiwoEnoch , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**